autoscale/src/main/java/com/sequenceiq/periscope/monitor/UpdateMonitor.java

```diff
@@ -0,0 +1,50 @@
+package com.sequenceiq.periscope.monitor;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.periscope.model.ScalingActivities;
+import com.sequenceiq.periscope.monitor.evaluator.EvaluatorExecutor;
+import com.sequenceiq.periscope.monitor.evaluator.update.ActivityUpdateEvaluator;
+import com.sequenceiq.periscope.service.ha.PeriscopeNodeService;
+
+@Component
+@ConditionalOnProperty(prefix = "periscope.enabledAutoscaleMonitors.update-monitor", name = "enabled", havingValue = "true")
+public class UpdateMonitor extends ScalingActivityMonitor {
+
+    // TODO: ID conflicts with CleanupMonitor?
```

Comment:
Verified this again. The `id` can be the same since this is differentiated based on the attached evaluator (and current timestamp) in `ExecutorServiceRegistry`, and `RejectedThread`.